Packs/ApiModules/Scripts/AnsibleApiModule/AnsibleApiModule.py

```diff
@@ -298,8 +302,14 @@ def generic_ansible(integration_name: str, command: str,
         if each_host_event['event'] in ["runner_on_ok", "runner_on_unreachable", "runner_on_failed"]:
 
             # parse results
-
-            result = json.loads('{' + each_host_event['stdout'].split('{', 1)[1])
+            str_to_parse = '{' + each_host_event['stdout'].split('{', 1)[1]
+            try:
+                result = json.loads(str_to_parse)
+            except JSONDecodeError as e:  # pragma: no cover
+                demisto.debug(e)
```

Comment:
please add some information such as:
```suggestion
                demisto.debug(e)
                demisto.debug('failed to parse response as JSON, will try to clean it from special characters and parse again')
```